src/components/Match/matchColumns.jsx

```diff
@@ -466,6 +466,31 @@ export const performanceColumns = [
       }
       return <div />;
     },
+  }, {
+    displayName: strings.th_others,
+    tooltip: strings.tooltip_others,
+    field: 'performance_others',
+    sortFn: true,
+    displayFn: (row, col, field) => {
+      const comp = [];
+      if (field) {
+        if (field.tracked_deaths) {
+          comp.push(<div>
+            { inflictorWithValue('bounty_hunter_track', abbreviateNumber(field.tracked_deaths), '',
+              `${field.tracked_deaths} ${strings.tooltip_others_total_gold} 
```

Comment:
Maybe put these two elements in an array and do `.join('\n')` so it's more apparent